docs/source/_sidebar.yaml

```diff
@@ -4,22 +4,39 @@ switcher:
     - label: v0
       latest: true
       href: ./
+defaultOpenDepth: 2
 items:
-  - label: "Welcome"
+  - label: "Overview"
     href: "."
   - label: "Quickstart"
     href: "./quickstart"
   - label: "Install"
     href: "./install"
-  - label: "User Guide"
-    href: "./guides"
+  - label: "Run the MCP Server"
+    href: "./run"
+  - label: "Define tools"
+    href: "./define-tools"
+  - label: "Configuration"
+    children:
+      - label: "YAML Config Reference"
+        href: "./config-file"
+      - label: "Custom Scalars"
+        href: "./custom-scalars"
+  - label: "Debugging"
+    href: "./debugging"
+  - label: "Deployment"
+    href: "./deploy"
   - label: "Authorization"
     href: "./guides/auth"
-  - label: "Command Reference"
-    href: "./command-reference"
+  - label: "Health Checks"
+    href: "./health-checks"
```

Comment:
Part of me thinks it would make sense to put Health Checks under the Configuration parent element, but I could go either way. Since the container is opened by default, it doesn't make it any less discoverable, unless the sidebar behavior changes.